<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editbook</class>
 <widget class="QDialog" name="editbook">
  <property name="windowTitle">
   <string>Edit Book</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
        background-color: #2c3e50;
    }
    QLabel {
        color: white;
        font-size: 16px;
    }
    QLineEdit {
        background-color: #ecf0f1;
        border: 1px solid #bdc3c7;
        border-radius: 5px;
        padding: 5px;
        font-size: 14px;
    }
    QPushButton {
        background-color: #27ae60;
        color: white;
        font-size: 16px;
        padding: 8px 20px;
        border-radius: 8px;
    }
    QPushButton:hover {
        background-color: #1e8449;
    }
   </string>
  </property>

  <layout class="QVBoxLayout" name="verticalLayout">

   <item>
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>ðŸ“• Book Name:</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLineEdit" name="lineEditName"/>
   </item>

   <item>
    <widget class="QLabel" name="labelPrice">
     <property name="text">
      <string>ðŸ’° Book Price:</string>
     </property>
    </widget>
   </item>

   <item>
    <widget class="QLineEdit" name="lineEditPrice"/>
   </item>

   <item>
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>ðŸ’¾ Save</string>
     </property>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
